### PR TITLE
Fix boolean filter

### PR DESCRIPTION
**Pull request by rpkilby**
Based off of #45 - can refactor/resubmit if necessary.

- Adds the BooleanWidget from django-filter to handle value casing.
- Adds tests to verify the expected behaviors from #6 and #25

### CODE REVIEW

File: tests/test_filterset.py [hunk 1]
```diff
@@ -767,8 +519,70 @@ def test_inset_char_filter(self):
...
+        # Uppercase False
+        GET = {'is_active': 'FALSE'}
+        filterset = UserFilter(GET, queryset=User.objects.all())
+        results = list(filterset)
+        self.assertEqual(len(results), 1)
+        self.assertEqual(results[0].username, 'user2')
+
+    def test_isnull_override(self):
+        self.assertIsInstance(
+            PersonFilter().filters['best_friend__isnull'],
```
Comment (rpkilby) [inline tests/test_filterset.py]: TODO: this isnull filter is not actually an overrides, since it's created as part of the RelatedFilter

**Comment by philipn** [comment]
Looks great! Thanks for the great work!

**Comment by rpkilby** [comment]
@philipn I think I need to do additional work here, not ready to merge yet.

**Comment by rpkilby** [comment]
@philipn Wouldn't hurt for you to look over this, but am going to go ahead and merge since it's pretty straightforward

**Pull request merged**